src/ctelnet.cpp

```diff
@@ -4084,6 +4084,7 @@ Some data loss is likely - please mention this problem to the game admins.)", co
                     // a beep every time the screen was refreshed!
                     // TODO: https://github.com/Mudlet/Mudlet/issues/5836 - provide option to actually make a (void) QApplication::beep() or a user-selected sound (different for each profile) and/or instead of the visual alert
```

Comment:
Perhaps you might want to amend this comment?

Or give the user the option to:
1. Not make a sound but just mark it with a pair of notes in the text as it did before this PR - if the "display control characters as" option is in the "default setting"
2. Make the standard beep as you are now going to do:
3. Make a custom - per-profile sound as the comment suggests.

Obviously this will need to be a setting that is saved and restored in the game save file.

Reply:
Back in #5573 discussions, @vadi2 pondered whether this comment should exist at all, so I did not touch it this time.

The list of ideas in your comment and in #5836 still mostly hold true. This is not fixing that issue.